src/DssSpell.sol

```diff
@@ -49,82 +121,191 @@ contract DssSpellAction is DssAction {
     //    https://ipfs.io/ipfs/QmVp4mhhbwWGTfbh2BzwQB9eiBrQBKiqcPRZCaAxNUaar6
     //
 
-    // --- Rates ---
-    // uint256 internal constant SEVEN_PT_FIVE_PERCENT_RATE = 1000000002293273137447730714;
+    function actions() public override {
+        // ----------------- Compound v2 D3M Onboarding -----------------
+        // https://vote.makerdao.com/polling/QmWYfgY2#poll-detail
+        {
+            VatAbstract vat = VatAbstract(DssExecLib.vat());
+            SpotAbstract spot = SpotAbstract(DssExecLib.spotter());
+            address daiJoin = DssExecLib.daiJoin();
+            address dai = DssExecLib.dai();
+            address vow = DssExecLib.vow();
+            address end = DssExecLib.end();
```

Comment:
These 4 are being used once so they could go inline as we are doing with `MCD_ADM`

Reply:
done